docs/source/tutorials/psij.ipynb

```diff
@@ -0,0 +1,548 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Psij Getting Started Tutorial"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Background\n",
+    "PSI/J (Portable Submission Interface for Jobs), is a Python abstraction layer which allows your HPC application to run virtually anywhere. <br> <br>\n",
+    "\n",
+    "<img src=\"./images/psij_overview.png\" width=\"350\"/>\n",
+    "<br>\n",
+    "<br>\n",
```

Comment:
`<br>` is not needed, but keep at least one empty line (`"\n"`), thus the following numbered list will be presented correctly (currently it is shown as a plain text)